### ISSUE TITLE
Vizor logo should link logged in users to their User Page

### ISSUE THREAD

**Issue by mattatgit** [vizorvr/vizor#1372]
If not logged in, it should go to the public page. If logged in, go to user page.
![screenshot 2016-02-29 16 58 22](https://cloud.githubusercontent.com/assets/6858525/13386916/b335ce0e-df05-11e5-8f80-9e8f3a7cdc3f.png)

**Comment by mattatgit** [comment]
This would still be a handy shortcut to have.

**Comment by esaruoho** [comment]
I agree, but also took a look at vizor/views/editor.handlebars which has this:
`<header id="editor-header" class="clearfix">
        <div class="editor-header clearfix">
                <div id="top-header-logo">
                        <a
                {{#if user.username}}
                    href="/{{user.username}}" title="Back to your profile page"
                {{else}}
                    href="/" title="Back to the homepage"
                {{/if}}
`


and it seems like it SHOULD show the /username but instead doesn't show. not sure why?

**Comment by esaruoho** [comment]
steps to "kinda" replicate with Safari
1. Enter http://patches.vizor.io/edit
2. See Vizor-logo links to Patches
3. Log in
4. See Vizor-logo links to Patches
5. Refresh page
6. Vizor-logo links to profile.

So, in essence, the Vizor-logo should be refreshed post-login, automatically. Not sure how hard that is. @gmarinov thoughts?

**Comment by mattatgit** [comment]
Related: the editor needs to have the Vizor logo remove and replaced with a Patches one.
